include/crm/msg_xml.h

```diff
@@ -60,6 +60,7 @@ extern "C" {
  */
 
 #define PCMK_META_ENABLED                   "enabled"
+#define PCMK_META_ATTR_EXPIRE               "expire_attrs"
```

Comment:
This should get its own "API:" commit (or just make that the commit message for this one)

Reply:
Fixed.